sdk/cosmos/azure-cosmos/samples/index_management.py

```diff
@@ -746,6 +746,61 @@ def get_embeddings(num):
         print("Entity doesn't exist")
 
 
+def use_full_text_policy(db):
+    try:
+        delete_container_if_exists(db, CONTAINER_ID)
```

Comment:
Deleting a container is happening async - espcially on multi-region account you probably need to add a dely to not introduce flakiness before trying to add a new container (or handle the 409 there and retry until new ocntainer can be created)

Reply:
this is just a snippet we use for samples, and every other method in this file does the same thing - I can go ahead and just change the id if you think there's an issue, but it's the general pattern we have across the file at the moment